src/test/java/edu/unc/lib/boxc/migration/cdm/services/ArchivalDestinationsServiceTest.java

```diff
@@ -47,6 +47,7 @@
 
 public class ArchivalDestinationsServiceTest {
     private static final String PROJECT_NAME = "proj";
+    protected final static String PROJECT_SOURCE = "cdm";
```

Comment:
Hopefully we can replace all these one off constants in tests with the central one suggested earlier